io/io/CMakeLists.txt

```diff
@@ -67,6 +71,10 @@ if(root7)
   target_sources(RIO PRIVATE v7/src/RFile.cxx)
 endif()
 
+if(uring)
+  target_link_libraries(RIO PRIVATE uring)
```

Comment:
This versus [that](https://github.com/root-project/root/pull/5919/files#diff-af3b638bc2a3e6c650974192a53c7291R229) should probably be rationalized?

Reply:
I kept this one and removed the other linker directives with 015977dfea